README.md

```diff
@@ -45,6 +45,44 @@ Then run the server
 python manage.py runserver_plus
 ```
 
+### Deploy to staging
```

Comment:
I think these changes fit better in the Manual - I think the README of this app should focus on this app, and not so much where it's deployed. This is especially the case with the Static and Teplates section - that stuff isn't part of this app, so I think it's up to the Maintenance Manual's development section to bring this app and static-budget-portal together. Do you think it's actually really important to have it here?